cookbooks/aws-parallelcluster-environment/resources/spack/partial/_spack_common.rb

```diff
@@ -15,10 +15,10 @@
 default_action :setup
 
 property :spack_user, String, required: false,
-         default: node['cluster']['cluster_user']
+         default: 'root'
```

Comment:
I don't think the spack user should be root.  Was there a specific reason you needed this?